lib/high_voltage/configuration.rb

```diff
@@ -25,17 +25,17 @@ def self.extended(base)
     end
 
     def action_caching=(value)
-      ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING)
+      ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING) unless value == false
```

Comment:
Looks like these are still using a trailing conditional. We look to avoid trailing conditionals like these because its a bit surprising and it's not immediately clear that the preceding code is only conditionally executed. Could you go through and convert these away from a trailing conditional?

For example:

```ruby
unless value == false
  ActiveSupport::Deprecation.warn(CACHING_DEPRECATION_WARNING)
end
```

This would also resolve the Hound warning around the line being to long.